core/src/main/java/com/datastax/dse/driver/internal/core/insights/PlatformInfoFinder.java

```diff
@@ -81,10 +81,23 @@ InsightsPlatformInfo getInsightsPlatformInfo() {
             this.getClass()
                 .getResourceAsStream("/com/datastax/dse/driver/internal/querybuilder/deps.txt"));
 
+    Map<String, RuntimeAndCompileTimeVersions> mapperProcessorDeps =
+        fetchDependenciesFromFile(
+            this.getClass()
+                .getResourceAsStream(
+                    "/com/datastax/oss/driver/internal/mapper/processor/deps.txt"));
+
+    Map<String, RuntimeAndCompileTimeVersions> mapperRuntimeDeps =
+        fetchDependenciesFromFile(
+            this.getClass()
+                .getResourceAsStream("/com/datastax/oss/driver/internal/mapper/deps.txt"));
+
     Map<String, Map<String, RuntimeAndCompileTimeVersions>> runtimeDependencies =
         new LinkedHashMap<>();
     putIfNonEmpty(coreDeps, runtimeDependencies, "core");
     putIfNonEmpty(queryBuilderDeps, runtimeDependencies, "query-builder");
+    putIfNonEmpty(mapperProcessorDeps, runtimeDependencies, "mapper-processor");
```

Comment:
Thanks for addressing this - did you validate that those dependencies are generated and picked by the insight client?

Reply:
I did by testing it against a local cluster and debug logs (didn't commit the debug logs). I tried creating a unit test, but the other tests around this mock a lot of the bits that happen here. I will try to see about an IT tests to verify it, unless you think it's not worth it.

Reply:
IT test may be error-prone because dependencies that are fetched can change in the future. Maybe we could have some kind of some test that only validates that `deps.txt` file was generated to alleviate future problems like this?

Reply:
I tried to address this here: https://github.com/datastax/java-driver/pull/1393#discussion_r374963189
@olim7t and @tomekl007, let me know what you think.